src/tests/EShopworld.WorkerProcess.UnitTests/Stores/CosmosDbLeaseStoreTests.cs

```diff
@@ -204,6 +204,110 @@ public async Task TestTryCreateLeaseAsyncConflict()
             result.Result.Should().BeFalse();
         }
 
+        [Fact, IsUnit]
+        public async Task TestAddLeaseRequestAsync_WhenAddingLeaseRequest_ShouldReturnTrue()
```

Comment:
For me there are at least following tests to be written for the added methods:
- add request, if document created, method should return true (I think this is what you intended to write here)
- the params passed to the method are properly transformed and passed as the document to be created
- in case of conflict, false is returned (you have it below)
- get winner, getting multiple requests, returns first one
- getting no requests, returns null

Reply:
- fixed selectWinner tests to cover both cases above

- checking that parameters passed to db are as expected